content/whats-new/Serenity-news.json

```diff
@@ -13,6 +13,13 @@
     "since-version": "Serenity",
     "link": "/pim/serenity/updates/2020-07.html#new-create-and-update-rules-from-the-pim"
   },
+  {
+    "title":"Attribute and option labels in product XLSX and CSV exports",
```

Comment:
```suggestion
    "title":"Attributes and options labels displayed in XLSX and CSV product exports",
```

Reply:
or "New option to display labels in the XLSX and CSV product exports"

Reply:
```suggestion
    "title":"Labels displayed in XLSX and CSV product exports",
```